circle.yml

```diff
@@ -4,96 +4,138 @@
-     # Android Support Repository, revision 35 / Local Maven repository for Support Libraries
-    - if [ ! -d "/usr/local/android-sdk-linux/extras/android/m2repository/com/android/support/design/24.1.0" ]; then echo y | android update sdk --no-ui --all --filter "extra-android-m2repository"; fi
+version: 2
+jobs:
+  build:
+    working_directory: ~/client
+    parallelism: 1
+    shell: /bin/bash --login
+    environment:
+      GOVERSION: 1.10.3
+
+      # Some node dependencies break without this.
+      CXX: g++-4.8
+
+    # TODO: Consider picking a better docker image.
+    #
+    # For more information on choosing an image (or alternatively
+    # using a VM instead of a container) see
+    # https://circleci.com/docs/2.0/executor-types/ To see the list of
+    # pre-built images that CircleCI provides for most common
+    # languages see https://circleci.com/docs/2.0/circleci-images/
+    docker:
+    - image: circleci/build-image:ubuntu-14.04-XXL-upstart-1189-5614f37
+      command: /sbin/init
+
+    steps:
+    - run:
+        name: Update PATH and Define Environment Variable at Runtime
+        command: |
+          echo 'export GOPATH=$HOME/go-path' >> $BASH_ENV
+          echo 'export PATH=$HOME/.yarn/bin:$GOROOT/bin:$PATH' >> $BASH_ENV
+
+          # Some node dependencies break without this.
+          echo 'export ANDROID_NDK=$ANDROID_HOME/ndk-bundle' >> $BASH_ENV
+          echo 'export ANDROID_NDK_HOME=$ANDROID_HOME/ndk-bundle' >> $BASH_ENV
+
+          source $BASH_ENV
+
+    - checkout
+
+    # Install node and java.
+    #
+    # TODO: Upgrade node?
+    - run: nvm install 8 && nvm alias default 8
+    - run: sudo update-alternatives --set java /usr/lib/jvm/jdk1.8.0/bin/java; sudo update-alternatives --set javac /usr/lib/jvm/jdk1.8.0/bin/javac; echo -e "export JAVA_HOME=/usr/lib/jvm/jdk1.8.0" >> $BASH_ENV
+
+    # Restore the dependency cache
+    - restore_cache:
+        keys:
+        # This branch if available
+        - v1-dep-{{ .Branch }}-
+        # Default branch if not
+        - v1-dep-master-
+        # Any branch if there are none on the default branch - this should be unnecessary if you have your default branch configured correctly
+        - v1-dep-
+
+    # - run: nvm ls-remote
+    - run: sudo apt-key adv --fetch-keys http://dl.yarnpkg.com/debian/pubkey.gpg
+    - run: echo "deb http://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+    - run: sudo apt-get update -qq
+    # - run: sudo apt-cache show yarn
+    - run: sudo apt-get install -y -qq yarn=1.0.2-1
+    - run: sudo rm -rf /usr/local/go*
+    - run: mkdir -p ~/downloads ~/go-path
+    - run: ls "$HOME/downloads"
+    - run: ls "$ANDROID_HOME"
+
+    - run: if [[ ! -e "$HOME/downloads/go.$GOVERSION.tar.gz" ]]; then wget -nv https://storage.googleapis.com/golang/go$GOVERSION.linux-amd64.tar.gz -O $HOME/downloads/go.$GOVERSION.tar.gz ; fi
```

Comment:
upgraded go to 1.10.3

Reply:
Let's not merge this until after the release